### PR TITLE
add dcs midnight and twilight version by matt3o

### PR DESCRIPTION
**Pull request by alinelena**
test-2

### CODE REVIEW

File: src/scss/dcs-abs.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+$color-dcs-abs-mid-alpha: #30312e;
```
Comment (noroadsleft) [inline src/scss/dcs-abs.scss:1]: This file isn't necessary; all these colors already exist in Configurator.

```
git rm src/scss/dcs-abs.scss
```

File: src/components/BaseKey.vue [hunk 2]
```diff
@@ -322,6 +322,7 @@ export default {
 @import '../scss/sp-pbt';
 @import '../scss/mt3-pbt';
 @import '../scss/mt3-abs';
+@import '../scss/dcs-abs';
```
Comment (noroadsleft) [inline src/components/BaseKey.vue:325]: The Signature Plastics / Pimp My Keyboard color list is at `src/scss/sp-abs.scss`.

```suggestion
```

File: src/components/colorways/index.js [hunk 3]
```diff
@@ -4,13 +4,14 @@ import sa from './sa';
...
 import mt3 from './mt3';
+import dcs from './dcs';
 import alphaCodes from './alphaCodes';
 import modCodes from './modCodes';
 
 export default {
   alphaCodes,
   modCodes,
-  list: [...dsa, ...sa, ...gmk, ...jtk, ...kat, ...mt3],
+  list: [...dsa, ...sa, ...gmk, ...jtk, ...kat, ...mt3, ...dcs],
```
Comment (noroadsleft) [inline src/components/colorways/index.js:14]: Putting DCS first will keep the list in alphabetical order.

```suggestion
  list: [...dcs, ...dsa, ...sa, ...gmk, ...jtk, ...kat, ...mt3],
```

File: src/scss/colorways.scss [hunk 4]
```diff
@@ -1570,3 +1570,36 @@
...
   // placeholder
 }
+
+.dcs-midnight-kb {
+  // placeholder
+}
+
+.dcs-midnight-key {
+  background: $color-dcs-abs-mid-alpha;
+  color: $color-dcs-abs-mid-accent-cyan;
```
Comment (noroadsleft) [inline src/scss/colorways.scss:1580]: The following suggestions swap out the color codes for the existing codes (they're the exact colors in each case).

```suggestion
  background: $color-sp-abs-GB;
  color: $color-sp-abs-VCO;
```

File: src/scss/colorways.scss [hunk 5]
```diff
@@ -1570,3 +1570,36 @@
...
+}
+
+.dcs-midnight-mod {
+  background: $color-dcs-abs-mid-mod;
+  color: $color-dcs-abs-mid-accent-cyan;
```
Comment (noroadsleft) [inline src/scss/colorways.scss:1585]:
```suggestion
  background: $color-sp-abs-GEW;
  color: $color-sp-abs-VCO;
```

File: src/scss/colorways.scss [hunk 6]
```diff
@@ -1570,3 +1570,36 @@
...
+}
+
+.dcs-midnight-accent {
+  background: $color-dcs-abs-mid-mod;
+  color: $color-dcs-abs-mid-accent-yellow;
```
Comment (noroadsleft) [inline src/scss/colorways.scss:1590]:
```suggestion
  background: $color-sp-abs-GEW;
  color: $color-sp-abs-YCH;
```

File: src/scss/colorways.scss [hunk 7]
```diff
@@ -1570,3 +1570,36 @@
...
+}
+
+.dcs-midnight-twilight-kb {
+  // placeholder
+}
+
+.dcs-midnight-twilight-key {
+  background: $color-dcs-abs-mid-alpha;
+  color: $color-dcs-abs-mid-accent-cyan;
```
Comment (noroadsleft) [inline src/scss/colorways.scss:1599]:
```suggestion
  background: $color-sp-abs-GB;
  color: $color-sp-abs-VCO;
```

File: src/scss/colorways.scss [hunk 8]
```diff
@@ -1570,3 +1570,36 @@
...
+}
+
+.dcs-midnight-twilight-mod {
+  background: $color-dcs-abs-mid-accent-cyan;
+  color: $color-dcs-abs-black;
```
Comment (noroadsleft) [inline src/scss/colorways.scss:1604]:
```suggestion
  background: $color-sp-abs-VCO;
  color: $qmk-black;
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Comment by noroadsleft** [comment]
Also will need a rebase on current `qmk:master`; this PR conflicts with the MT3 LOTR/3277 merge.

**Comment by alinelena** [comment]
i messed the rebase... will be faster to read the stuff in a clean pr.

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**